TaskfMRIAnalysis/TaskfMRIAnalysis.sh

```diff
@@ -284,7 +289,8 @@ then
 		--lvl1fsfs=$LevelOnefsfNames \
 		--lvl2task=$LevelTwofMRIName \
 		--lvl2fsf=$LevelTwofsfName \
-		--summaryname=$SummaryName \
+		--summarydir=$SummaryDir \
+        --summaryprefix=$SummaryPrefix \
```

Comment:
tabs/spaces mismatch

Reply:
Also, if someone doesn't read the help, or just forgets `--summaryprefix` when using `--summarydir`, it looks like at present it will use a literal NONE along with the provided `--summarydir`, and `makeSubjectTaskSummary.sh` appears to use it as-is in that case.